packages/firestore/src/api/database.ts

```diff
@@ -146,6 +147,7 @@ const DEFAULT_HOST = 'firestore.googleapis.com';
 const DEFAULT_SSL = true;
 const DEFAULT_TIMESTAMPS_IN_SNAPSHOTS = true;
 const DEFAULT_FORCE_LONG_POLLING = false;
+const DEFAULT_FORCE_AUTO_DETECT_LONG_POLLING = true;
```

Comment:
Please set this to false and use https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/test/integration/util/helpers.ts#L160 to pass custom settings.

Reply:
Should I set this for both the emulator and prod settings in https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/test/integration/util/settings.ts#L35 or should I be passing this in differently?

Reply:
This is the only place you need to set it to false. What you have now applies to all settings, unless it is overridden by the user.